src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/SinkIslandCheck.java

```diff
@@ -71,7 +72,10 @@ public SinkIslandCheck(final Configuration configuration)
     @Override
     public boolean validCheckForObject(final AtlasObject object)
     {
-        return this.validEdge(object) && !this.isFlagged(object.getIdentifier());
+        return this.validEdge(object) && !this.isFlagged(object.getIdentifier())
+                && HighwayTag.highwayTag(object)
```

Comment:
At this point, we can assume the object is an Edge, so you could avoid the `orElse` by using the Edge's `highwayTag()` function.
```java
((Edge) object).highwayTag().isMoreImportantThan(this.minimumImportanceHighway)
```